chapters/gamestructure.adoc

```diff
@@ -97,6 +97,8 @@ The <<Referee, referee>> tosses a coin with both <<Robot Handler, robot handlers
 ==== Choosing Keeper Id
 The <<Referee, referee>> asks both <<Robot Handler, robot handlers>> which robot they will use as the keeper and forwards this information to the <<Game Controller Operator, game controller operator>>.
 
+The keeper Id can also be changed during game play, this can be done automatically by using the <<Game Controller, game controller>> network interface or manually, in this case, the <<Game Controller Operator, game controller operator>> is responsible for checking if the ball is in a valid position and making the change.
```

Comment:
it is a bit unclear, what the last sentence is referring to. What about: `interface or by asking the GCO to change it in the GC. The GCO must not change the keeper id until the ball is at a valid position.`